public/i18n/en-us.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
public/i18n holds client translations and is handled by a gulp task
/i18n holds server translations and is handled by i18n-node during invocation